api/seeds/20-example-document.ts

```diff
@@ -0,0 +1,70 @@
+import * as Knex from "knex";
+
+exports.seed = async function(knex: Knex) {
+  const document = {
+    id: "mrXRm8gYI",
+    name: "Example Document",
+    version: 0,
+  };
+  await knex("documents").insert(document);
+
+  const func = {
+    id: "3rX3ND6f9M",
+    document_id: document.id,
+    path: "/Ability Modifier",
```

Comment:
Is the `path` value going to be used anywhere that spaces will cause a problem?

Reply:
Nope, for the purposes of the app it's just a sequence of unicode strings delimited by `/`.